tests/prod/prod_sanity_suite_test.go

```diff
@@ -116,6 +146,16 @@ func TestCloudRouterCreate_DIGP(t *testing.T) {
 	terraform.InitAndApply(t, terraformOptions)
 	output := terraform.Output(t, terraformOptions, "cloud_router_id")
 	assert.NotNil(t, output)
+
+	terraform.Apply(t, terraformOptions)
```

Comment:
This shouldn't need to be here because it is already being done earlier in method with `InitAndApply`. Why is the redundant `Apply` in place?

Reply:
Sorry for the confusion, Updated the PR with latest changes.